apps/backend/src/http-ping/status-change/http-ping-status-change.service.ts

```diff
@@ -0,0 +1,64 @@
+import { Injectable } from '@nestjs/common';
+import { OnEvent } from '@nestjs/event-emitter';
+import { HttpPingEvent } from '../events/http-ping-event.enum';
+import { HttpPingCreatedEvent } from '../events/definitions/http-ping-created.event';
+import { PingStatus } from './enum/ping-status.enum';
+import { RedisService } from '../../shared/redis/redis.service';
+import { CommunicationChannelMessagingService } from '../../communication-channel/messaging/communication-channel-messaging.service';
+import { HttpMonitorReadService } from '../../http-monitor/read/http-monitor-read.service';
+
+@Injectable()
+export class HttpPingStatusChangeService {
```

Comment:
As you noticed, this is actually http monitor responsibility not the atomic http ping.